### DISCUSSION TITLE
[Q regarding multiple schemas] Is ConnectionPool state shared across requests ?

### DISCUSSION THREAD

**Discussion by VaZark** [SeaQL/sea-orm#1676] [Q&A]
I'm currently using `django-pgschemas` for my multi-tenant SaaS product where the db connection is modified to select the active schema for each request. We can then proceed with the standard application logic. 

This doc snippet tells me I can use whatever schemas I like. However, if I change the pg schema during runtime in one request, would it affect all other requests since it changes of the connection state  ? Or does it not affect the pool but only the particular connection

```rust
let connect_options = ConnectOptions::new("postgres://root:root@localhost/database".into())
    .set_schema_search_path("my_schema".into()) // Override the default schema
    .to_owned();
```

**Comment by billy1624** [comment]
Hey @VaZark, I don't recommend you changing the search path at runtime. You can create a connection pool (`DbConn`) per tenant. Then, you can switch the `DbConn` for multi-tenant application.
